crates/milli/src/vector/error.rs

```diff
@@ -67,7 +67,7 @@ pub enum EmbedErrorKind {
     #[error("could not authenticate against {embedding} server{server_reply}{hint}", embedding=match *.1 {
         ConfigurationSource::User => "embedding",
         ConfigurationSource::OpenAi => "OpenAI",
-        ConfigurationSource::Ollama => "ollama"
+        ConfigurationSource::Ollama => "Ollama"
```

Comment:
Do we need to inform the gui about this change?

Reply:
hmmm, I'm not sure, it looks pretty minor?